tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/smt/SolverConfig.scala

```diff
@@ -7,15 +7,16 @@ package at.forsyte.apalache.tla.bmcmt.smt
  * @param debug Enable the debug mode (activated with --debug). Write the log file to log%d.smt.
  * @param profile Enable the profile mode (activated with --profile). Report the metrics.
  * @param randomSeed The random seed to be passed to z3 as :random-seed.
+ * @param smtEncoding The SMT encoding to be used.
  *
  * @author Igor Konnov
```

Comment:
```suggestion
 * @author Igor Konnov, Rodrigo Otoni
```
```suggestion
 * @author Igor Konnov
```

Reply:
I didn't know if making such a change to a file warranted being added as an author.

Reply:
It's a tiny file. We don't have precise rules about authorship. It's like if you make substantial changes to the code, which in this case applies :)